io_directory_example/settings/lens_system3_config.yml

```diff
@@ -0,0 +1,47 @@
+---
+system_name: 'SL2SJ0219-0829'
```

Comment:
`system_name` should say `'lens_system3'`.

Reply:
Thanks for spotting that